### PR TITLE
feat: platform services

### PR DESCRIPTION
**Pull request by shuplenkov**

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -3,7 +3,9 @@ const config = {
     port: 3000,
   },
   grpc: {
-    nativePort: 3010,
```
Comment (antouhou) [inline src/config.js]: Why did we remove the native port? Did we decide to not use native connection anymore?

File: src/index.js [hunk 2]
```diff
@@ -311,23 +277,6 @@ class DAPIClient {
    */
   getUserByName(username) { return this.makeRequestToRandomDAPINode('getUser', { username }); }
```
Comment (antouhou) [inline src/index.js]: Hm, maybe we want to rename this one to `getIdentityByName` or something like that too?

Comment (shumkov) [reply src/index.js]: We are going to remove unused methods in forthcoming PR

File: src/index.js [hunk 3]
```diff
@@ -311,23 +275,6 @@ class DAPIClient {
...
-
-    const nodeToConnect = await this.MNDiscovery.getRandomMasternode();
-
-    const client = new CorePromiseClient(`${nodeToConnect.getIp()}:${this.getGrpcPort()}`);
-
-    return client.updateState(updateStateRequest);
-  }
-
   // Here go methods that used in VMN. Most of this methods will work only in regtest mode
   searchUsers(pattern, limit = 10, offset = 0) { return this.makeRequestToRandomDAPINode('searchUsers', { pattern, limit, offset }); }
```
Comment (antouhou) [inline src/index.js]: I guess we need to delete that one too then

**Pull request merged**